store-front/Dockerfile

```diff
@@ -1,18 +1,15 @@
 # note: expected build context is the git root directory
-FROM node:18-alpine
+FROM node:16-alpine
```

Comment:
Is there a reason for switching to nodejs version 16, instead of 18? Otherwise probably best to stick with nodejs version 18.

Reply:
Fixed now